### PR TITLE
updates one human law board

### PR DESCRIPTION
**Pull request by utahclock**
Fixes Issue brought up here, https://forums.yogstation.net/index.php?threads/20thgirl-supernoodles13.11986/#post-114139

#### Changelog

:cl:
Tweak: Adds more definition to One Human Law Set Board.
/:cl:

### CODE REVIEW

**Comment by Super3222** [comment]
I think he used a **traitor board** instead of a one human board...

File: code/game/objects/items/weapons/AI_modules.dm [hunk 1]
```diff
@@ -165,7 +165,7 @@ AI MODULES
 	if(!targName)
 		return
 	targetName = targName
-	laws[1] = "Only [targetName] is human"
+	laws[1] = "Only [targetName] is human. Do not reveal who is human"
```
Comment (ShadowDeath6) [inline code/game/objects/items/weapons/AI_modules.dm]: Might want to make this two separate laws and have proper punctuation

**Comment by utahclock** [comment]
@Super3222 He specifically used the ONE HUMAN board in the AI core which gives 0th law. He didnt use the hacked AI traitor board as that gives ### (Iconic Law)

**Comment by ghost** [comment]
no reason, an ai should know better

File: code/game/objects/items/weapons/AI_modules.dm [hunk 2]
```diff
@@ -158,14 +158,14 @@ AI MODULES
 	name = "'OneHuman' AI Module"
 	var/targetName = ""
 	origin_tech = "programming=4;materials=4"
-	laws = list("Only SUBJECT is human.")
+	laws = list("Only SUBJECT is human. Do not reveal who is human.")
```
Comment (ShadowDeath6) [inline code/game/objects/items/weapons/AI_modules.dm]: I still think this should be reworded to say "do not reveal them", as the way it's worded right now sounds like a second law

**Pull request closed without merging**